### ISSUE TITLE
pod check ignores namespaces in project

### ISSUE THREAD

**Issue by yoshi314** [Napsty/check_rancher2#10]
I wanted to check pods in ingress-nginx namespace in System, but the script would check entire System project and gave me the totals from System.

I have made a small adjustment to the script : 

```
if [ $namespacename != "" ]; then
        EXTRAURL="?namespaceId=$namespacename"
fi
  api_out_pods=$(curl -L -s ${selfsigned} -u "${apiuser}:${apipass}" "${proto}://${apihost}/v3/project/${projectname}/pods$EXTRAURL")
```

Apparently you can pass ?namespaceId=somenamespace to api call to filter out by namespace.

**Comment by Napsty** [comment]
Cool, thanks for this hint @yoshi314 . This will help to solve #7

**Comment by yoshi314** [comment]
i checked it back then, and it worked fine.

**Comment by Napsty** [comment]
fixed in 1.4.0.
